Sources/swift-utilities/CommandLineAction.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CommandLineAction.swift
+//  
+//
+//  Created by Bill Gestrich on 12/8/21.
+//
+
+import Foundation
+
```

Comment:
To make this all a little more clear, how about something like this:

- invocationName: String
- shortInvocationName: String (if you want to support both full/abbreviated)

That may help to disambiguate the difference between these things and "name". The name is a bit of misnomer -- that was really more of a command description.

Reply:
Changing `name` to `longName` and adding `shortName` works. Thanks

Reply:
Done